cli/cli.ts

```diff
@@ -1525,16 +1540,20 @@ function buildSemanticUIAsync() {
 }
 
 function updateDefaultProjects(cfg: pxt.TargetBundle) {
-    let templatesRoot = path.join("libs", "templates");
-    let defaultProjects = [
+    let defaultTemplates = [
         "blocksprj",
         "tsprj"
     ];
 
-    defaultProjects.forEach((p) => {
-        let projectPath = path.join(templatesRoot, p);
+    getTemplates().forEach((templatePath) => {
+        if (defaultTemplates.indexOf(path.basename(templatePath)) === -1) {
```

Comment:
My idea was to support more offline HEX images than just the default core + radio. With this in place, it's super easy to create more HEX images: just add a new project under `libs/templates/`, and that will result automatically in a 2nd HEX file in the hexcache. If we remove this check, then we can't do that, since any project under `libs/templates/` will end up in target.json.

What do you think?